<ui version="4.0" >
 <class>DSExportDialog</class>
 <widget class="QDialog" name="DSExportDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="fileLabel" >
       <property name="text" >
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameInput" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fileNameButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item rowspan="2" row="0" column="0" >
      <widget class="QListWidget" name="sensorList" />
     </item>
     <item row="0" column="1" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="startLabel" >
         <property name="text" >
          <string>Start Index:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="startInput" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="startInfoLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>xx.xx.xxxx - yy:yy</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="stopLabel" >
         <property name="text" >
          <string>End Index:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="stopInput" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="stopInfoLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>xx.xx.xxxx - yy:yy</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="separatorLabel" >
         <property name="text" >
          <string>Separator:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="separatorInput" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>1000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>;</string>
         </property>
         <property name="maxLength" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="separatorHintLabel" >
         <property name="text" >
          <string>(for CSV only)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="infoLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="text" >
        <string>info</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DSExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DSExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
